src/webgpu/api/validation/render_pipeline/depth_stencil_state.spec.ts

```diff
@@ -32,6 +32,45 @@ g.test('format')
     t.doCreateRenderPipelineTest(isAsync, !!info.depth || !!info.stencil, descriptor);
   });
 
+g.test('depthCompare')
+  .desc(`The depthCompare in depthStencilState is optional for stencil-only formats.`)
+  .params(u => u.combine('isAsync', [false, true]).combine('format', kTextureFormats))
+  .beforeAllSubcases(t => {
+    const { format } = t.params;
+    const info = kTextureFormatInfo[format];
+    t.skipIfTextureFormatNotSupported(format);
+    t.selectDeviceOrSkipTestCase(info.feature);
+  })
+  .fn(t => {
+    const { isAsync, format } = t.params;
+    const info = kTextureFormatInfo[format];
+    const depthWriteEnabled = !!info.depth;
+    const descriptor = t.getDescriptor({
+      depthStencil: { format, depthWriteEnabled },
+    });
+
+    t.doCreateRenderPipelineTest(isAsync, !info.depth && !!info.stencil, descriptor);
```

Comment:
!(info.depth && depthCompare === undefined). Then we'd always set the depthWriteEnabled to make sure we aren't testing this.

Reply:
Got it